packages/common/src/api/tan-query/comments.ts

```diff
@@ -68,19 +69,12 @@ const messages = {
  * QUERIES
  *
  */
-export type GetCommentsByUserIdArgs = {
-  userId: ID
-  currentUserId?: ID
-  sortMethod?: CommentSortMethod
-  pageSize?: number
-}
-
-export const useGetCommentsByUserId = ({
-  userId,
-  currentUserId,
+export const useUserComments = (
+  userId: ID | null,
```

Comment:
nit: A lot of the other modern tan-query hooks use an object for params instead of positional ones. Should probably follow that pattern? (`{userId, pageSize}`)